src/structures/CommandContext.ts

```diff
@@ -54,7 +54,7 @@ export class CommandContext {
    */
   public ran: string
 
-  constructor (opts: { worker: Worker, message: APIMessage, command: CommandOptions, prefix: string, ran: string, args: string[] }) {
+  constructor (opts: {worker: Worker, message: APIMessage, command: CommandOptions, prefix: string, ran: string, args: string[]}) {
```

Comment:
You might want to disable your autoformatter, or at least go back and fix the changes.